src/pages/TodoList/reducers/index.js

```diff
@@ -0,0 +1,63 @@
+import { createSlice } from "@reduxjs/toolkit";
+import { v4 as uuid } from "uuid";
+
+const initialState = {
+  tasks: [],
+};
+
+const todosSlice = createSlice({
+  name: "todoList",
+  initialState,
+  reducers: {
+    createTask: (state, { payload: taskData }) => {
+      const newTask = {
+        id: uuid(),
+        title: taskData.taskTitle,
+        text: taskData.taskText,
+        isCompleted: false,
+        isEditMode: false,
+      };
+
+      state.tasks.unshift(newTask);
+      return state;
+    },
+    removeTask: (state, { payload: taskId }) => {
+      const taskIndex = state.tasks.findIndex(({ id }) => id === taskId);
+      state.tasks.splice(taskIndex, 1);
+      return state;
```

Comment:
you don't need to return state here. In redux toolkit you mutate state directly.
fix it here and in the all same places